.prettierrc.yml

```diff
@@ -1,3 +1,4 @@
 tabWidth: 4
 printWidth: 100
 trailingComma: "none"
+endOfLine: "auto"
```

Comment:
I'm not 100% sure this does the same as the git option. According to the git docs, `text=auto` should make sure that only LF's go into the repository, while according to [Prettier's docs](https://prettier.io/docs/en/options#end-of-line) it maintains existing line endings, but might change them if in a file with mixed endings. Wouldn't it be better to set it to `"lf"` here?

Also, if Git makes sure that only `lf` enters the repository, does it still matter what Prettier does?
I'm also asking this because at some point soon-ish, I intend to replace Prettier, and I don't know yet how its replacement will behave in this specific situation.

Reply:
Indeed, Prettier's auto mode is not the same. Thanks for bringing this to my attention. I read the doc, but skipped this part.  
To be honest, for Windows systems, it is quite a pain that Prettier is enforcing the line endings. This should be the responsibility of the git attributes.  
For this project we can probably get away with the suggestion of Prettier:

> If you want to make sure that your entire git repository only contains Linux-style line endings in files covered by Prettier:
> 
> 1. Ensure Prettier’s endOfLine option is set to `lf` (this is a default value since v2.0.0)
> ...
> 1. Add `* text=auto eol=lf` to the repo’s .gitattributes file. You may need to ask Windows users to re-clone your repo after this change to ensure git has not converted LF to CRLF on checkout.
> 1. All modern text editors in all operating systems are able to correctly display line endings when \n (LF) is used. However, old versions of Notepad for Windows will visually squash such lines into one as they can only deal with \r\n (CRLF).


There are project's where the line endings must match properly, especially with script files. But I think we don't have any issues here.  
I'll adapt the PR accordingly.

Reply:
See other thread.